src/constants.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  DEPRECATED_DEFAULT_CONFIG_FILE_PATH: 'webpack.haul.js',
   DEFAULT_CONFIG_FILE_PATH: 'haul.config.js',
```

Comment:
Future improvement: change these constants to `[DEPRECATED_]DEFAULT_CONFIG_FILENAME`, as it's not a path and it's actually a bit misleading while reading code in `getWebpackConfigPath.js`

Reply:
Agree. I'm going to change it right now.